src/darwin/Framework/CHIP/CHIPOnOff.h

```diff
@@ -19,16 +19,17 @@
 #define CHIP_ONOFF_H
 
 #import "CHIPDevice.h"
+#import "CHIPDeviceCallback.h"
 #import <Foundation/Foundation.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface CHIPOnOff : NSObject
 
-- (nullable instancetype)initWithDevice:(CHIPDevice *)device;
-- (BOOL)lightOn;
-- (BOOL)lightOff;
-- (BOOL)toggleLight;
+- (nullable instancetype)initWithDevice:(CHIPDevice *)device endpoint:(uint8_t)endpoint;
```

Comment:
add callbackQueue:(dispatch_queue_t) callbackQueue to initialiser instead

Reply:
done